targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -41,7 +41,7 @@ final case class CoordinatesForm(
   target:           TargetResult,
   searchAndGo:      SearchCallback => Callback,
   goToRaDec:        Coordinates => Callback,
-  onNameChange:     NonEmptyString => Callback
+  onNameChange:     NonEmptyString => IO[Unit]
```

Comment:
`IO` composes more easily with the setters in `View`, `Undo`, GraphQL clients. Besides, in the caller it was originally an `IO` that had been converted to `Callback` and now I needed to convert it back here to `IO` to compose with `View.set`.

Should we convert the other callbacks to `IO`? In general I find it more ergonomic to work with `IO`s and convert them to `Callback` at the last moment, which sometimes is not even necessary if we end up passing a `View` to a final control (like `FormInputEV`).

Reply:
Sound good, it would be nice to work with only one `Effect` but I guess that requires big changes in `scalajs-react`

Reply:
Would it be OK with you to switch to `IO` internally and only convert to `CB` when interfacing with `scalajs-react` (via `.runInCB` ?

Reply:
Yes that would be fine